Dockerfile

```diff
@@ -6,15 +6,16 @@ RUN sudo bash -c "echo \"deb http://ftp.us.debian.org/debian testing main contri
   && sudo apt-get update \
   # bzip2 and libgconf-2-4 are necessary for extracting firefox and running chrome, respectively
   && sudo apt-get install bzip2 libgconf-2-4 node-less cmake build-essential clang-format-6.0 \
-                  uglifyjs chromium ccache libncurses6 gfortran f2c \
+                  uglifyjs chromium ccache libncurses6 gfortran f2c swig \
```

Comment:
Yes, please make a separate PR with the changes in Dockerfile. We might also want to update to `circleci/python:3.7.0-stretch` image to a latter python 3.7 release if there is one.